Cargo.lock

```diff
@@ -540,7 +558,7 @@ version = "0.2.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "94b38784f05c5e908bb8751e9b9f29fbcd470f636c0d0a76a0f90c0c823f3b68"
 dependencies = [
- "clap 4.0.30",
+ "clap 3.2.23",
```

Comment:
This should have gone to 4.10 but cargo has a personal vendetta against me. I would submit a proper pull request but I'm on my phone.